.gitignore

```diff
@@ -22,3 +22,5 @@ erl_crash.dump
 # Ignore package tarball (built via "mix hex.build").
 telepoison-*.tar
 
+# Temporary files, for example, from tests.
+/tmp/
```

Comment:
why? which test adds temporary files? (is it just a best practice of yours? :D)

Reply:
This was from the latest `.gitignore` file created from `mix new` of the most recent Elixir version.
See https://github.com/elixir-lang/elixir/blob/master/lib/mix/lib/mix/tasks/new.ex#L306